cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -81,6 +80,17 @@ spec:
                   values:
                   - rook-ceph-osd-prepare
               topologyKey: kubernetes.io/hostname
+        # topologySpreadConstraints:
```

Comment:
What happens if the constraints are specified and it is an older version of k8s that doesn't support the feature or the quality gate isn't enabled? Will the constraints just be ignored or would it cause a failure? If it is just ignored, perhaps we should just enable this example by default.

Reply:
I confirmed that the Rook cluster using topologySpreadConstraints in K8s v1.17 (topogySpreadConstrains is not enabled by default) didn't cause any problem. Here K8s just ignore this field. So I un-commented the configuration of this feature in the example.